.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,74 @@
+#
```

Comment:
This is OK, but these checks should be distributed inside json-semantic-playground or a similar, external, repository.

This allows a central distribution and versioning of all the hooks that repositories have to use.